extensions/amp-gwd-animation/0.1/validator-amp-gwd-animation.protoascii

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
FYI, validator rules are now at the extension level, not version level. Please move up one directory.

Reply:
Done.